### PR TITLE
Added various options + moved color rules to "general features"

### PR DESCRIPTION
**Pull request by federico-trotta**

### CODE REVIEW

File: _mydocs/calendar-pro/options/granularity/group.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ date:               2022-07-27
 modified:           2022-07-27
 order:              /calendar-pro/options/calendar-granularity/group-by
 internal:           group
-draft:              true
 ---
-**Default value:** Auto
\ No newline at end of file
+The **Group** parameter manages how data are grouped in the visual.
+
+The **Default value** is **Auto**.
```
Comment (danieleperilli) [inline _mydocs/calendar-pro/options/granularity/group.md]: Please leave **Default value:** XXX
We used this label everywhere for other visuals (e.g. Bullet Chart options).
Can you please revert this change here and in other places where you changed it?

Comment (federico-trotta) [reply _mydocs/calendar-pro/options/granularity/group.md]: Shure! Working on that right now. :)
